tests/QMLEngine/scope.js

```diff
@@ -30,4 +30,18 @@ describe('QMLEngine.scope', function() {
     var childB = contextVariable(qml, "childB");
     expect(childB.value).toBe(4);
   });
+
+  it('local var assigns doesnt kill properties', function() {
+    var qml = load('SafeFromLocals', this.div);
+    expect(qml.a).toBe("333");
+    expect(qml.b).toBe("b_init_value");
+    expect(qml.c).toBe( "" );
```

Comment:
Unwanted spacing.
Do not bother fixing it, I will fix it when I will be landing this.